pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -206,6 +206,23 @@ def test_OA_quadratic_strategy(self):
                               [TerminationCondition.optimal, TerminationCondition.feasible])
                 self.assertAlmostEqual(
                     value(model.objective.expr), model.optimal_value, places=1)
+    
+    @unittest.skipUnless(SolverFactory('appsi_cplex').available(), 
+                         "APPSI_CPLEX not available.")
+    def test_OA_APPSI_solver(self):
+        """Test the outer approximation decomposition algorithm."""
+        with SolverFactory('mindtpy') as opt:
+            for model in model_list:
+                results = opt.solve(model, strategy='OA',
+                                    mip_solver='appsi_cplex',
+                                    nlp_solver=required_solvers[0],
+                                    integer_to_binary=True
```

Comment:
This test is checking both APPSI and the integer to binary transformation. Let's check only APPSI here as the following test takes care of the transformation.